pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/LiteralTransformFunction.java

```diff
@@ -86,6 +86,10 @@ public double getDoubleLiteral() {
     return _doubleLiteral;
   }
 
+  public float getFloatLiteral() {
```

Comment:
If this PR is to fix backward incompatibility issue, why bother adding this method?

Reply:
because fixing backward incompatibility needs this new API..